vue-frontend/src/components/partials/NewHeader.vue

```diff
@@ -77,8 +77,43 @@
                   >{{ navbar.name }}</a
                 >
               </li>
+              <div class="tw-mt-1" @click="showMenu = !showMenu">
+                <button type="button" aria-label="hamburger">
+                  <svg
```

Comment:
Should we show this icon, if services and resources are not available on live website?